primitives/src/v1.rs

```diff
@@ -454,6 +454,76 @@ impl PoV {
 	}
 }
 
+/// SCALE and Zstd encoded [`PoV`].
+#[derive(Clone, Encode, Decode, PartialEq, Eq)]
+pub struct CompressedPoV(Vec<u8>);
```

Comment:
I don't believe this should be in `primitives` as it's not needed in the runtime. Maybe `node/primitives`.

Reply:
Although `PoV` is not needed in the runtime either, we do all validation off chain. Shall I move both?

Reply:
Either way, if it is ok with you, I would open up another PR for that as I am working already on another PR, which would profit from this being merged already.

Reply:
Ok, please move this in a follow-on. `PoV` can be in the `node/primitives` as well, it seems. Thanks for observing that.